homeassistant/components/pi_hole/binary_sensor.py

```diff
@@ -25,16 +30,18 @@ class PiHoleBinarySensor(PiHoleEntity, BinarySensorEntity):
     """Representation of a Pi-hole binary sensor."""
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the sensor."""
         return self._name
 
     @property
-    def unique_id(self):
+    def unique_id(self) -> str:
         """Return the unique id of the sensor."""
         return f"{self._server_unique_id}/Status"
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return if the service is on."""
-        return self.api.data.get("status") == "enabled"
+        if self.api.data.get("status") == "enabled":
+            return True
+        return False
```

Comment:
Right, in this case I think the best option is explicitly ignore it. If at some point `hole` package will become typed, mypy will suggest to delete this ignore statement. Current solution is some hack around fact that underlying package is not typed.
```suggestion
       return self.api.data.get("status") == "enabled"  # type: ignore[error-name]
```

Reply:
I think to ignore as above is the way to go for now.